src/dmd/initsem.d

```diff
@@ -95,6 +95,11 @@ extern(C++) Initializer initializerSemantic(Initializer init, Scope* sc, ref Typ
 {
     Type t = tx;
 
+    static Initializer err()
+    {
+        return new ErrorInitializer();
```

Comment:
(maybe out of the scope of this PR but) We have `ErrorExp.get()` as a singleton instance, why not `ErrorInitializer.get()` too?

Reply:
A good idea, but for another PR.